third_party/pcsc-lite/naclport/server_clients_management/src/clients_manager.cc

```diff
@@ -136,7 +137,6 @@ void PcscLiteServerClientsManager::Detach() {
   typed_message_router_->RemoveRoute(&create_handler_message_listener_);
   typed_message_router_->RemoveRoute(&delete_handler_message_listener_);
   DeleteAllHandlers();
-  pp_instance_ = nullptr;
```

Comment:
nit: `global_context_ = nullptr`

Reply:
Ack - It's intentional that we're keeping `global_context_`, since this object is intended to be available even after "detaching". The idea is that the consumer can still call methods of this class, which simplifies the consumer's implementation, it's just that the `PostMessageToJs()` will return an error if used after detaching.